z2/src/setup.rs

```diff
@@ -546,6 +546,7 @@ impl Setup {
                 state_rpc_limit: state_rpc_limit_default(),
                 failed_request_sleep_duration: failed_request_sleep_duration_default(),
                 enable_ots_indices: false,
+                max_rpc_response_size: max_rpc_response_size_default(),
```

Comment:
can have sense having in the config file a `max_rpc_response_size_in_mb` or `max_rpc_response_size_in_kb`?

@JamesHinshelwood ?

Reply:
Happy to leave it as is, or do it either way.

Reply:
no strong opinion my side @shawn-zil

Reply:
@JamesHinshelwood commented to leave it as Bytes (during the call-chat today). So, I'll leave it as-is and mark this comment as closed @frankmeds